site.cfg.example

```diff
@@ -52,6 +52,28 @@
 #       True) to tell numpy.distutils to prefer static libraries (.a) over
 #       shared libraries (.so). It is turned off by default.
 #           search_static_first = false
+#
+#   runtime_library_dirs/rpath
+#       List of directories that contains the libraries that should be 
+#       used at runtime, thereby disregarding the LD_LIBRARY_PATH variable.
+#       See 'library_dirs' for formatting on different platforms.
+#           runtime_library_dirs = /opt/blas/lib:/opt/lapack/lib
+#       or equivalently
+#           rpath = /opt/blas/lib:/opt/lapack/lib
+#
+#   extra_compile_args
+#       Add additional arguments to the compilation of sources.
+#       Simple variable with no parsing done. 
+#       Provide a single line with all complete flags.
+#           extra_compile_args = -g -ftree-vectorize
```

Comment:
it is not clear here what the scope of these arguments are, library/runtime/include-dir are all related to binaries, compile args apply to sources. Does it apply to all numpy sources like the OPT env variable) or only part of them?
I don't see a reason this flag should be specific to sections concerning libraries, maybe having one global one is enough?